bionic/dagviz.py

```diff
@@ -4,16 +4,53 @@
 is also required.
 """
 
+import warnings
+from pathlib import Path
 from collections import defaultdict
-from io import BytesIO
+from io import BytesIO, IOBase
 
-from .optdep import import_optional_dependency
+from .optdep import import_optional_dependency, oneline
 module_purpose = 'rendering the flow DAG'
 hsluv = import_optional_dependency('hsluv', purpose=module_purpose)
 pydot = import_optional_dependency('pydot', purpose=module_purpose)
 Image = import_optional_dependency('PIL.Image', purpose=module_purpose)
 
 
+class FlowImage:
+    def __init__(self, pydot_graph):
+        '''
+        Given a pydot graph object, create Pillow Image or SVG represented as XML text.
```

Comment:
It would probably be more useful to have a class-level docstring, targeted at the end user, which explains what this class is and how you're expected to use it.

Reply:
Just to expand on this previous comment: it's more useful to have a docstring for the entire class than for the constructor, since Bionic users will never need to construct this class themselves. Maybe something like this?

```
class FlowImage:
    """
    An image visualizing a Bionic flow. This object provides a API similar to a
    Pillow Image, but supports an additional `svg` format in addition to the
    Pillow formats. You can use ``save`` to write this image to disk or
    ``show`` to display the image in a UI. This object will also be
    automatically rendered in a Jupyter notebook as an SVG image.
    """
```